src/app/pages/CreateConnectorPage/StepConfigurator.tsx

```diff
@@ -45,17 +45,20 @@ const ConnectedCustomConfigurator: FunctionComponent<{
     )
   );
   if (duplicateMode) {
-    const clearedConfiguration = clearEmptyObjectValues(configuration);
     const defaultEntries = JSON.parse(JSON.stringify(connectorData?.connector));
     let combineConfiguration = {};
     if (configuration instanceof Map) {
       combineConfiguration = _.merge(
         {},
         defaultEntries,
-        mapToObject(clearedConfiguration)
+        clearEmptyObjectValues(mapToObject(configuration))
```

Comment:
Wonder, why `_.merge` here btw?  Isn't this the same:

```
combineConfiguration = { ...defaultEntries, ...clearEmptyObjectValues(mapToObject(configuration)) };
```

values coming from the latter spread operator will override values from the previous spread operator.  Just curious.  Anyways, totally didn't spot the logic change implied when I adjusted that function but see it now, sorry!  Will make it up to you when I get back from PTO with an integration test that covers the duplicate flow, deal?  :-)